ethrex/src/cli.rs

```diff
@@ -32,11 +32,46 @@ pub fn cli() -> Command {
                 .value_name("PORT")
                 .action(ArgAction::Set),
         )
+        .arg(
+            Arg::new("addr")
```

Comment:
it is not clear to me what this address means? Maybe we should prefix it like we do with the other ones, like authrpc, discovery, etc..

Reply:
I'm not sure what prefix should I put on it, maybe @MegaRedHand can give us some insight

Reply:
The naming was taken from geth/reth. It's used for both discovery and rlpx (`discovery.addr` only overrides the discovery part).

Reply:
I like `p2p.addr`